### ISSUE TITLE
In Version 1.6.0 of Lightning, WandB UI excludes the full command string, contrary to previous versions of Lightning

### ISSUE THREAD

**Issue by amorehead** [PyTorchLightning/pytorch-lightning#12594]
## 🐛 Bug

In Version 1.6.0 of Lightning, WandB excludes the full command string, contrary to previous versions of Lightning

### To Reproduce

Run a Lightning training job that logs uses WandB.

### Expected behavior

WandB, with Lightning, should display the full command string used to run the script serving as the entry point for the training job.

### Additional context

Compared to version 1.5.10 of Lightning, in version 1.6.0 Lightning now only displays the name of the entry-point script used to kick off the training job, in the WandB web UI:

1.5.10:
![image](https://user-images.githubusercontent.com/7051982/161446781-53431e81-6431-4c72-9cfb-5212689c22de.png)

1.6.0:
![image](https://user-images.githubusercontent.com/7051982/161446811-5eb9e9e4-672e-4dbf-b4a6-7a2abceebf5b.png)

**Comment by manangoel99** [comment]
Hello! I'm an engineer at Weights & Biases. Could you also please share the version of wandb that you're using

**Comment by tshu-w** [comment]
@manangoel99 I'm facing the same issue with wandb 0.12.11

**Comment by manangoel99** [comment]
Cool! Thanks for letting me know. Will check and get back to you

**Comment by amorehead** [comment]
Indeed. This issue occurs with PyTorch Lightning 1.6.0 and WandB 0.12.11, for me at least.

**Comment by amorehead** [comment]
@awaelchli,

Thanks for your quick reply. If I am understanding correctly, for us Lightning users, does this mean we need to wait for a patch to WandB to see this issue resolved?

**Comment by manangoel99** [comment]
@amorehead The bug has been fixed in wandb and will be available in the next release

**Comment by awaelchli** [comment]
@amorehead Yes, or we need to call the API differently, not sure. Downgrading the wandb version to < 0.12.10 would solve your issue temporarily.

**Comment by kptkin** [comment]
@amorehead the fix is already in master, so if you want to install `wandb` from  master:
`pip install git+https://github.com/wandb/client.git`
it should just work

**Comment by amorehead** [comment]
Thank you all for your hard work and quick replies! I think the issue has been identified and solved.
